app/views/users/index.html.haml

```diff
@@ -26,15 +26,15 @@
     %thead
       %tr
         %th
-          = sort_link(@q, :last_name, 'Name',{}, {title: 'Sort on Name'})
+          = sort_link(@q, :last_name, 'Name', {}, {title: 'Sort on Name'})
         %th
-          = sort_link(@q, :email,{}, {title: 'Sort on Email'})
+          = sort_link(@q, :email, {}, { title: 'Sort on Email' })
         %th
-          = sort_link(@q, :type,{}, {title: 'Sort on Type'})
+          = sort_link(@q, :type_nulls_last, t('activerecord.attributes.user.type'), {}, { title: 'Sort on User Type' })
```

Comment:
this should never happen as type is required for user creation, but still probably a good check.

Reply:
We have quite a few unconfirmed users in the membership site. Would not have caught this had I not had an unconfirmed test user locally.